### ISSUE TITLE
Build is failing for Hikey 960 when using master branch

### ISSUE THREAD

**Issue by rounaka** [OP-TEE/build#311]
I checked out the manifest from the master branch (commit f0e10fb67a701a0f736ad027798da7c25057532b)
`repo init -u https://github.com/OP-TEE/manifest.git -m hikey960.xml -b master`
Then I ran the build
`make all`
The build failed with the following output at the end :

```bash
make[1]: Entering directory '/home/rounaka/optee/out-br'
umask 0022 && make -C /home/rounaka/optee/buildroot O=/home/rounaka/optee/out-br/. all
/usr/bin/make -j1 O=/home/rounaka/optee/out-br HOSTCC="/usr/bin/gcc" HOSTCXX="/usr/bin/g++" silentoldconfig
  GEN     /home/rounaka/optee/out-br/Makefile
\>>> toolchain-external-custom  Configuring
Incorrect selection of kernel headers: expected 4.18.x, got 4.6.x
package/pkg-generic.mk:219: recipe for target '/home/rounaka/optee/out-br/build/toolchain-external-custom/.stamp_configured' failed
make[2]: *** [/home/rounaka/optee/out-br/build/toolchain-external-custom/.stamp_configured] Error 1
Makefile:16: recipe for target '_all' failed
make[1]: *** [_all] Error 2
make[1]: Leaving directory '/home/rounaka/optee/out-br'
common.mk:201: recipe for target 'buildroot' failed
make: *** [buildroot] Error 2
```

**Comment by rounaka** [comment]
Thanks @jforissier . I did run make toolchains. But because of the way the makefile toolchain.mk is written to check for presence of toolchains it failed to identify that I have version 6.x of toolchains which I had downloaded for building 3.2.0 branch. I have deleted the toolchains folder so that running make toolchains will download toolchain 8.x. I will rebuild with this toolchain and see if this solves the issue.
